style.css

```diff
@@ -0,0 +1,186 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  background-color: #191765;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+#top-navigation {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: auto;
+  height: 48px;
+  margin-top: 60px;
+}
+
+.name {
+  color: #fff;
+  text-decoration: none;
+}
+
+.fa {
+  color: #fff;
+}
+
+.text {
+  color: #fff;
+  text-decoration: none;
+}
+
+.profile {
+  padding-left: 24px;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 16px;
+}
+
+#side-nav {
+  width: 33.333%;
+  display: flex;
+  justify-content: flex-end;
+  color: #fff;
+  padding-right: 40px;
+}
+
+.fa-bars {
+  font-size: 30px;
+  width: 33.33%;
+}
+
+.header h1 {
+  text-align: center;
+  font-weight: 700;
+  color: #7d73fc;
+  line-height: 72px;
+  padding-top: 50px;
+}
+
+.header h2 {
+  text-align: center;
+  font-weight: 500;
+  color: #edecff;
+  line-height: 44px;
+  text-size-adjust: 32px;
+  padding-top: 67;
+}
+
+.header p {
+  margin-top: 20px;
+  font-weight: 400;
+  text-align: center;
+  color: #c8c8d4;
+  line-height: 28px;
+  width: 400px;
+  height: 120px;
+  padding-left: 80px;
```

Comment:
**[REQUIRED CHANGES]**

- Hey kindly use `padding: 24px` to fix the issue with the paragraph text here below:  


**your design**

![center](https://user-images.githubusercontent.com/59397159/170582177-c73c6503-e750-444f-8d9f-75eb233fe23f.png)

**Figma design**

![figcenter](https://user-images.githubusercontent.com/59397159/170582422-70ed99fa-f3d0-4e47-88aa-c136ff1126c6.png)